packages/toolpad-app/src/utils/useLocalStorageState.ts

```diff
@@ -67,7 +67,7 @@ export default function useLocalStorageState<V>(
   const storedValue: V = React.useSyncExternalStore(
     (cb) => subscribe(key, cb),
     () => getSnapshot(key) ?? initialValue,
-    () => initialValue,
+    () => getSnapshot(key) ?? initialValue,
```

Comment:
Is there any con to using the value in storage if it already exists?
Starting the value as `null` when it already exists was giving me some problems as we're now storing the latest app when we create it.